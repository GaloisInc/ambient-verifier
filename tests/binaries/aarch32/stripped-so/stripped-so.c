#include "getone.h"

void nothing() {}

int main() {
  if (getone()) {
    nothing();
  }
  return 0;
}
