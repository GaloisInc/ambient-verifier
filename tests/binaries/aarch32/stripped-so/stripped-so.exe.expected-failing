successful: 1
failed: 0
sharedObjectsDir: tests/binaries/aarch32/stripped-so
entryPointAddr: 0x5a0
