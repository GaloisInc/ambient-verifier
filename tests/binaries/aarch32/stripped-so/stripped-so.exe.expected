successful: 0
failed: 0
sharedObjectsDir: tests/binaries/aarch32/stripped-so
entryPointAddr: 0x5a0
